client/src/app/videos/+video-watch/video-share.component.html

```diff
@@ -12,12 +12,34 @@ <h4 class="modal-title">Share</h4>
       <div class="modal-body">
         <div class="form-group">
           <label>URL</label>
-          <input #urlInput (click)="urlInput.select()" type="text" class="form-control input-sm readonly" readonly [value]="getVideoUrl()" />
+          <div class="input-group">
+            <input #urlInput (click)="urlInput.select()" type="text" class="form-control input-sm readonly" readonly [value]="getVideoUrl()" />
+            <div class="input-group-btn" placement="bottom right">
+              <button [ngxClipboard]="urlInput" (click)="activateCopiedMessage()" type="button" class="btn btn-default btn-search">
```

Comment:
For the icon, I suggest using `glyphicon-copy` (https://getbootstrap.com/docs/3.3/components/#glyphicons)